src/game.js

```diff
@@ -0,0 +1,69 @@
+function Game(frames = 10) {
+  this._frames = [];
+  this._scoreLog = [];
+  this._maxFrames = frames;
+  this._allThePins = 10;
+}
+
+Game.prototype.addFrame = function(frame) {
+  this._checkIfGameOver();
+  this._frames.push(frame);
+};
+
+Game.prototype.checkAllScores = function() {
+  return this._frames;
+};
+
+Game.prototype.checkGameLength = function() {
+  return this._maxFrames;
+};
+
+Game.prototype.calculateGameTotal = function() {
```

Comment:
Love the clarity of this method! To answer your question, I think the helpers you've created really aid the clarity of the code (eg lines 41 - 51).